polus-precompute-volume-plugin/src/requirements.txt

```diff
@@ -0,0 +1,7 @@
+filepattern==1.4.4
+numpy==1.19.5
+bfio==2.0.6
```

Comment:
Upgrade to filepattern 1.4.6 and bfio 2.1.2. This will add support for Zarr as an input data type.